pkg/container-utils/testutils/options.go

```diff
@@ -52,6 +55,12 @@ func WithImage(image string) Option {
 	}
 }
 
+func WithImageTag(tag string) Option {
```

Comment:
What is this useful for? Can't we pass the tag within the image name instead (`WithImage("foo:latest")`)?

Reply:
Intent was an easy way to switch tags without having to deconstruct the image string